renderdoc/driver/vulkan/wrappers/vk_cmd_funcs.cpp

```diff
@@ -1127,8 +1130,10 @@ bool WrappedVulkan::Serialise_vkEndCommandBuffer(SerialiserType &ser, VkCommandB
 
         ObjDisp(commandBuffer)->EndCommandBuffer(Unwrap(commandBuffer));
 
-        if(m_Partial[Primary].partialParent == BakedCommandBuffer)
-          m_Partial[Primary].partialParent = ResourceId();
+        // TODO: preserve so that m_RenderState can be updated at the end
```

Comment:
I marked a TODO mainly to call out that I am doing it here, if it sounds good, I can remove

Reply:
I _think_ this should be OK. I believe originally this was necessary to handle the case of command buffers being submitted twice or recorded multiple times in a frame, but looking at the code now I don't think it should be necessary.